pom.xml

```diff
@@ -135,6 +135,16 @@
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
     </properties>
 
+    <dependencyManagement>
```

Comment:
All the `org.slf4j` dependencies can be removed from the main pom.xml, what do you think ?

Reply:
Funny enough, since we're using log.info in the springboot module's integration tests, the logback-test in the core module was actually saving our asses.
I added a slf4j-simple dependency in the springboot module with the 'test' scope to avoid impacting users